cpp/autosar/test/rules/M0-1-4/test.cpp

```diff
@@ -30,4 +30,12 @@ void test_templates() {
   f1<B>(); // Triggers a NON_COMPLIANT case in f1(), because B is a POD type
   f1<C>(); // Does not trigger a NON_COMPLIANT case in f1(), because C is not a
            // POD type
-}
\ No newline at end of file
+}
+
+class C1 {
+  static constexpr int unused{1}; // NON_COMPLIANT
+  static constexpr int used{2};   // COMPLIANT
+  int test_use() { return used; }
+  static constexpr int size{3};               // COMPLIANT[FALSE_POSITIVE]
```

Comment:
I am not seeing this false positive. Can we remove the [FALSE_POSITIVE] label?

Reply:
Good spot - I added the test and the label, and forgot to remove it. Thanks!
```suggestion
  static constexpr int size{3};               // COMPLIANT
```